event-based-simulators/main/cumulocityAPI.py

```diff
@@ -7,19 +7,28 @@
 
 MOCK_REQUESTS = os.environ.get('MOCK_C8Y_REQUESTS') or 'false'
 
-user_and_pass_bytes = base64.b64encode((C8Y_TENANT + "/" + C8Y_USER + ':' + C8Y_PASSWORD).encode('ascii')) # bytes
-user_and_pass = user_and_pass_bytes.decode('ascii') # decode to str 
+user_and_pass_bytes = base64.b64encode((C8Y_TENANT + "/" + C8Y_USER + ':' + C8Y_PASSWORD).encode('ascii'))  # bytes
+user_and_pass = user_and_pass_bytes.decode('ascii')  # decode to str
```

Comment:
please revert unnecessary changes, so the PR doesnt get cluttered with irrelevant changes...